server/src/main/java/org/bonitasoft/web/rest/server/utils/JacksonSerializerHelper.java

```diff
@@ -36,21 +36,14 @@ public JacksonSerializerHelper() {
         numberTypes.add(Double.class.getCanonicalName());
     }
 
-    protected String getStringValue(Serializable value) {
-        if (value == null) {
-            return null;
-        }
-        return String.valueOf(value);
-    }
-
-    protected void writeValueAndStringValue(JsonGenerator jgen, String fieldName, Serializable value) throws IOException {
+    protected void writeValueAndStringValue(final JsonGenerator jgen, final String fieldName, final Serializable value) throws IOException {
         jgen.writeObjectField(fieldName, value);
-        if (numberTypes.contains(value.getClass().getCanonicalName())) {
-            jgen.writeObjectField(fieldName + "_string", getStringValue(value));
+        if (value != null && numberTypes.contains(value.getClass().getCanonicalName())) {
```

Comment:
I agree. having the attribute always present makes the API easier to use, I think.